Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7Parser201305.java

```diff
@@ -383,12 +384,10 @@ public static String extractTelecom(PRPAMT201306UV02ParameterList params) {
                 TELExplicit telecomValue = patientTelecom.getValue().get(0);
                 LOG.info("Found patientTelecom in query parameters = " + telecomValue.getValue());
                 telecom = telecomValue.getValue();
-                if (telecom != null) {
-                    if (!telecom.startsWith("tel:")) {
-                        // telecom is not valid without tel: prefix
-                        telecom = null;
-                        LOG.info("Found patientTelecom in query parameters is not in the correct uri format");
-                    }
+                if (NullChecker.isNotNullish(telecom) && !telecom.startsWith("tel:")) {
```

Comment:
Optional: use StringUtils.startWith() method from apache common language. It will check nullvalue as well.